nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -194,7 +194,7 @@ process get_software_versions {
     echo $workflow.nextflow.version > v_nextflow.txt
     fastqc --version > v_fastqc.txt
     multiqc --version > v_multiqc.txt
-    scrape_software_versions.py &> software_versions_mqc.yaml
+    python scrape_software_versions.py &> software_versions_mqc.yaml
```

Comment:
Adding `python` wont work because the `scrape_software_versions.py` script wont be found.

Reply:
I had something like this somewhere in the back of my head - thanks for confirming 👍

Reply:
We should make the `tools` repo actually try to run a pipeline.

Can you explain why @drpatelh? Are you sure it's not just an R thing? Have you tested it?

Reply:
```
python: can't open file 'scrape_software_versions.py': [Errno 2] No such file or directory
```

Goddam, it's annoying when you're right 😆 

But I thought that the `bin` directory was always on the `PATH`? Is that not the point? Does that not also work for script names?

Reply:
Sorry, was just getting to this...unfortunately, the script names arent found if `python` or `R` are used as a prefix unless you provide the full path to the script (or are running in the same directory as the script I think). The `bin/` is exported into `$PATH` but this only works if the script name is the first argument. There is probably a better, low-level voodoo magic explanation but I cant seem to find a good resource to explain this in more detail :)

Reply:
To fix it I think it works to add:
```
    input:
    path 'scrape_software_versions.py' from "$baseDir/bin/scrape_software_versions.py"
```